compiler/src/dotty/tools/dotc/transform/patmat/Space.scala

```diff
@@ -22,6 +22,7 @@ import reporting.diagnostic.messages._
 import reporting.trace
 import config.Printers.{exhaustivity => debug}
 import util.SourcePosition
+import NullOpsDecorator._
```

Comment:
It would be good if Fengyun could review this part.

Reply:
Fengyun ok'ed the `Space` changes.